nautobot/extras/apps.py

```diff
@@ -44,3 +44,10 @@ def convert_field_to_list_tags(field, registry=None):
         from nautobot.extras.health_checks import DatabaseBackend
 
         plugin_dir.register(DatabaseBackend)
+
+        # Register netutils jinja2 filters
```

Comment:
Not clear to me that this needs to be done inside an app `ready()` function. This code might be a better fit somewhere like `nautobot/utilities/utils.py`?

Reply:
I think it's important to register these filters when the Nautobot is starting, as far as I know I don't think they will be loaded automatically in `nautobot/utilities/utils.py`

Reply:
I suggested that file as that's where the `render_jinja2` function is defined - any code that consumes these filters should be using that function as its renderer, generally speaking.